libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -144,6 +146,14 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("OPTIONS", 9,   AP_Arming, _arming_options, 0),
 
+    // @Param: MAGTHRESH
+    // @DisplayName: Compass magnetic field strength error threshold vs earth magnetic model
+    // @Description: Compass magnetic field strength error threshold vs earth magnetic model.  X and y axis are compared using this threhold, Z axis uses 2x this threshold.  0 to disable check
+    // @Units: mgauss
```

Comment:
```suggestion
    // @Units: mGauss
```

Reply:
Cool thanks.

I've also changed the new parameter to an AP_Int16 (it was an AP_Float).